test-apps/display-test-app/src/frontend/ClassificationsPanel.ts

```diff
@@ -66,11 +74,54 @@ export class ClassificationsPanel extends ToolBarDropDown {
     this._realityModelListDiv = IModelApp.makeHTMLElement("div", { parent: this._element });
     this._modelListDiv = IModelApp.makeHTMLElement("div", { parent: this._element });
     this._propertiesDiv = IModelApp.makeHTMLElement("div", { parent: this._element });
+    this._realityModelPickerMenu = createNestedMenu({
+      label: "Reality Model Picker",
+    });
 
     this._element.style.display = "block";
     this._element.style.cssFloat = "left";
     this._element.style.width = "max-content";
     this._element.style.minWidth = "350px";
+
+    this._element.appendChild(this._realityModelPickerMenu.div);
+  }
+
+  private async populateRealityModelsPicker(): Promise<void> {
+    this._realityModelPickerMenu.div.style.display = "none";
+    clearElement(this._realityModelPickerMenu.body);
+
+    const view = this._vp.view;
+    const ecef = this._vp.iModel.ecefLocation;
+    if (!view.isSpatialView() || undefined === ecef) {
+      return;
+    }
+
+    const range = new CartographicRange(this._vp.iModel.projectExtents, ecef.getTransform());
+    let available = new Array<ContextRealityModelProps>();
+    try {
+      if (this._iTwinId !== undefined) {
+        if (IModelApp.authorizationClient && IModelApp.authorizationClient.hasSignedIn) {
+          const accessToken = await IModelApp.authorizationClient.getAccessToken();
+          available = await new RealityDataAccessClient().queryRealityData(accessToken, { iTwinId: this._iTwinId, range });
+        }
+      }
+    } catch (_error) {
+      // eslint-disable-next-line no-console
+      console.error("Error in query RealitydataList, you need to set SVT_STANDALONE_SIGNIN=true, and are your SVT_ITWIN_ID and IMJS_BUDDI_RESOLVE_URL_USING_REGION correctly set?");

```

Comment:
The prefix has changed from SVT_ to IMJS_ on master.

Reply:
Updated.